clang/docs/checkedc/3C/INSTALL.md

```diff
@@ -0,0 +1,54 @@
+# Building 3C
+
+This document describes our current setup and build process for 3C.  You may find helpful information about topics not addressed here in the [general Checked C build documentation](../Setup-and-Build.md).  Some items here are potentially applicable to Checked C as a whole, and we hope to eventually harmonize them with the Checked C documentation as appropriate.
+
+## Supported environments
+
+We work primarily on Linux.  Checked C supports Windows, but we do not test 3C regularly on Windows.  Some of us use Mac OS X, which Checked C does not officially support; we have gotten 3C to mostly work with some additional steps (see below), but there are some known problems and [Microsoft is still considering whether to work with us to address them](https://github.com/microsoft/checkedc/issues/424#issuecomment-720648442).
+
+## Basics
+
+**(TODO: Test from start to finish!)**
+
+Assuming you have already cloned the `checkedc-clang` repository, run the following (from the root of the working tree) for a basic build:
+
+```
+# See https://github.com/correctcomputation/checkedc-clang/issues/296 about this.
+(llvm/projects/checkedc-wrapper && git clone https://github.com/microsoft/checkedc)
+
+mkdir build && cd build
+cmake ../llvm -G Ninja -DLLVM_ENABLE_PROJECTS=clang
+ninja TARGET
+```
+
+where TARGET stands for the target you want to build.  For the `3c` command-line tool, the target is `3c`.  Executables will end up in `build/bin`.
```

Comment:
Should point out that they should build `clang` too, assuming they want the Checked C compiler. No sense in separately installing Microsoft's repository and ours and building in two separate places.

Reply:
Done.